exp-player/addon/templates/components/exp-exit-survey.hbs

```diff
@@ -78,9 +78,9 @@
             </div>
         </div>
         <hr>
-        <div class="form-group">
+        <!--<div class="form-group">
```

Comment:
@kimberscott can I uncomment this block since the `-pilot` version of this component seems to be handling this?